# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/coreutils/hotfix.patch.musl
# Copyright (C) 2021 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- coreutils-9.0/src/ls.c.vanilla	2021-10-09 23:18:16.244303461 +0200
+++ coreutils-9.0/src/ls.c	2021-10-09 23:20:06.139303416 +0200
@@ -2273,7 +2273,7 @@
       if (linelen < 0)
         {
           /* Suppress bogus warning re comparing ws.ws_col to big integer.  */
-# if __GNUC_PREREQ (4, 6)
+# if 0
 #  pragma GCC diagnostic push
 #  pragma GCC diagnostic ignored "-Wtype-limits"
 # endif
@@ -2282,7 +2282,7 @@
               && 0 <= ioctl (STDOUT_FILENO, TIOCGWINSZ, &ws)
               && 0 < ws.ws_col)
             linelen = ws.ws_col <= MIN (PTRDIFF_MAX, SIZE_MAX) ? ws.ws_col : 0;
-# if __GNUC_PREREQ (4, 6)
+# if 0
 #  pragma GCC diagnostic pop
 # endif
         }
